tests/Rule/EnsureExplicitNullableTypesTest.php

```diff
@@ -69,17 +73,21 @@ public static function invalidProvider(): iterable
             'function foo(?int $foo = null, int $bar = null, int $baz)',
             'function foo(int|string|null $foo = null, int $bar = null, int $baz)',
             'int $foo = null,',
+            'private int $foo = null',
```

Comment:
This is not valid in PHP 8.* 

https://3v4l.org/B1AF9#v8.1.28 

That's why i only checked arguments here

Reply:
Ok, makes sense 👍